lighthouse-core/scripts/update-report-fixtures.js

```diff
@@ -31,7 +50,7 @@ async function update() {
     url,
   ].join(' ');
   const flags = cliFlags.getFlags(rawFlags);
-  await cli.runLighthouse(url, flags, undefined);
+  await cli.runLighthouse(url, flags, budgetedConfig);
```

Comment:
since we don't run `bin.js` (which would load from a `--budget-path`), this could either put a budget on `flags.budgets` or make a config as it does here. The config seemed cleaner, but I'm not attached to the approach.

Reply:
config sgtm 👍